debian/control

```diff
@@ -0,0 +1,42 @@
+Source: eos-data-distribution
+Section: python
+Priority: standard
+Maintainer: Philip Withnall <withnall@endlessm.com>
+Build-Depends:
+ debhelper (>= 10),
+ dh-python,
+ python-all,
+ python-pyndn,
+ python-pytest,
+ python-setuptools,
+ python-setuptools-git,
+ python3-all,
+ python3-pyndn,
+ python3-setuptools,
+ python3-setuptools-git,
+Standards-Version: 3.9.7
+Testsuite: autopkgtest-pkg-python
+X-Python-Version: >= 2.7
+X-Python3-Version: >= 3.0
+
+Package: python-eosdatadistribution
+Architecture: all
+Depends:
+ ${python:Depends},
+ ${misc:Depends},
+Description: NDN producers and daemons (Python 2)
+ Named Data Networking provides name-based access to data over the
+ network.
+ .
+ This package contains the services and library for Python 2.
+
+Package: python3-eosdatadistribution
```

Comment:
we need to split that into more packages i think, at least:
 - lib
 - dbus service
 - ostree-store
 - usb stuff